app/helpers/promos_helper.rb

```diff
@@ -80,4 +80,45 @@ def on_channel_type(channel)
       raise
     end
   end
+
+  def reporting_interval_column_header(reporting_interval)
+    case reporting_interval
+    when "by_day"
```

Comment:
I'd prefer `by_day`, `by_week`, `by_month`, `cumulative` to be stored as constants. It's easy to misspell and get wrong (i.e. the convention is daily, weekly, monthly, running_total)

Reply:
Yeah, BD asked for 'by week' etc, but I will change since I think that's better.